cloudbuild.yaml

```diff
@@ -0,0 +1,24 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+# For more information about Image pushing refer to https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/README.md
+timeout: 3600s
+
+options:
+  substitution_option: ALLOW_LOOSE
+
+steps:
+  - name: 'gcr.io/cloud-builders/bazel'
+    env:
+    - IMAGE_REGISTRY=gcr.io
+    - IMAGE_REPO=k8s-image-staging
```

Comment:
Is this the image repo we want to keep using? I had the impression we want to use `k8s-staging-cloud-provider-gcp`?

Reply:
I'm culto cargoing with the things I find, so no idea what is best